api/envoy/config/core/v3/address.proto

```diff
@@ -164,6 +164,10 @@ message BindConfig {
   // :ref:`extra_source_addresses <envoy_v3_api_field_config.core.v3.BindConfig.extra_source_addresses>`
```

Comment:
And we should update the comment for the `extra_source_addresses` field at line 153. To clarify the only two addresses allowed is default behavior when `local_address_selector_name` isn't specified.